pkgs/applications/misc/snowsql/default.nix

```
@@ -1,49 +1,20 @@
-{ lib
-, stdenv
-, fetchurl
-, rpmextract
-, patchelf
-, makeWrapper
-, openssl
-, libxcrypt-legacy
-}:
-
-stdenv.mkDerivation rec {
+{
+  stdenv,
```

Given that this package isn't built, but rather installs from a pre-built binary use of stdenvNoCC is likely preferable.
```suggestion
  stdenvNoCC,
```